client/src/components/pages/RoomShowPage.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react"
+import { io } from "socket.io-client"
+import ChatWindow from "../chat/ChatWindow"
+
+const RoomShowPage = ({ user, params, url }) => {
+  const [socket] = useState(io("http://localhost:3000"))
```

Comment:
later on, we may have to add some conditional logic here to ensure this is updated for production